apps/Dataman/src/Utils/colors.ts

```diff
@@ -0,0 +1,61 @@
+export const getColorsByLetter = (letter: string) => {
```

Comment:
why we need this file?

Reply:
Why is this color behind profile needed or useful?

Reply:
This file has been removed now